config.py

```diff
@@ -1,4 +1,11 @@
+# Twitter API keys
+
 consumer_key = ""
 consumer_secret = ""
 access_token_key = ""
 access_token_secret = ""
+
+# Telegram API keys
+
+token = ""
+chat_id = ""
```

Comment:
can you name the variable a bit more verbose? 

Something like telegram_token and things like that?

Reply:
sure